site/docs/v1/tech/apis/_standard-post-response-codes.adoc

```diff
@@ -15,19 +15,27 @@ ifndef::success_code[]
 |The request was successful. The response will contain a JSON body.
 endif::[]
 
+
+ifndef::error_malformed[]
```

Comment:
I understand why you did this (so you didn't have to update everywhere this is included!), but I'd prefer one of two other options:

Either use a double negative:

`ifndef::omit_error_malformed[]`

Or use `ifdef` and add it to all the other places this is included to be more consistent wit the other if statements, such as ifdef::missing_message:

`ifdef::error_malformed[]`